Sources/Benchmarks/main.swift

```diff
@@ -13,11 +13,16 @@
 // limitations under the License.
 
 import Benchmark
+import PenguinStructures
 
-Benchmark.main([
-  arrayStorage,
-  nonBlockingCondition,
-  nonBlockingThreadPool,
-  adjacencyList,
-  parallelExpander,
-])
+Benchmark.main(
```

Comment:
Nit: consider reverting the formatting?

Reply:
Swift-format will do it my way.  I know because I wrote the rule: any opening delimiter not matched by a close on the same line is the last thing on the line, except for closure signatures and comments.